installing_invest_python_package.md

```diff
@@ -0,0 +1,56 @@
+# Installing the InVEST python package
+
+## Suggested method
+
+**Pattern:**
+```
+conda create -y -n <name> python=<python version>
+conda activate <name>
+conda install -y gdal=<gdal version>
+pip install natcap.invest[ui]==<invest version>
+```
+Replace `<name>` with any name you'd like to give your environment.
+Replace `<python version>` with a python version known to be compatible with the desired invest version.
+Replace `<gdal version>` with a GDAL version known to be compatible with the desired invest version.
+Replace `<invest version>` with the desired invest version.
+
+Most of the time, it is not really necessary to specify the versions of `python`, `gdal`, and `natcap.invest`. If you do not specify a version, the latest version will be installed. Usually the latest versions are compatible with each other, but not always. Specifying versions that are known to work can prevent some problems.
+
```

Comment:
It could be useful to point to `requirements.txt` here, for the specific version compatibilities. If we add this content directly to the API docs, that requirements list would already be there.

Reply:
I added a sentence linking to requirements.txt. This made me realize we don't document which versions of python are supported... they only exist in our GHA workflows. If a python version doesn't belong in requirements.txt, maybe in the README?